src/tiny_slam/tiny_world.h

```diff
@@ -15,17 +15,20 @@
 
 struct TinyWorldParams {
   double localized_scan_quality, raw_scan_quality;
+  const double SIG_XY;
+  const double SIG_TH;
+  const double BAD_LMT;
+  const double TOT_LMT;
+  const double HOLE_WIDTH;
+
+  TinyWorldParams(double const_params[]): SIG_XY(const_params[0]),
```

Comment:
What is the function that maps a member to an array's index? Why have the current order been selected?